dataset-spec/README.md

```diff
@@ -0,0 +1,85 @@
+| temporal_extent | string                                | Temporal extent (required)      | Temporal extent covered by the dataset. Date/time intervals MUST be formatted according to ISO 8601. Open date ranges are not supported by ISO 8601 and MUST be encoded as proposed by [Dublin Core Collection Description: Open Date Range Format](http://www.ukoln.ac.uk/metadata/dcmi/date-dccd-odrf/2005-08-13/). |
+| links           | [Link Object]                         | Links (required)                | A list of references to other documents, see Link Object for further documentation. TODO: Remove if catalog is revised. |
+
+### Provider Object
+
+| Element | Type   | Name                  | Description                                     |
+| ------- | ------ | --------------------- | ----------------------------------------------- |
+| name    | string | Organization name     | The name of the organization or the individual. |
+| url     | string | Organization homepage | Homepage of the provider.                       |
+
+###  Host Object
+
+| Element        | Type    | Name                  | Description                                                  |
+| -------------- | ------- | --------------------- | ------------------------------------------------------------ |
+| description    | string  | Description           | Detailed description to explain the hosting details. [CommonMark 0.28](http://commonmark.org/) syntax MAY be used for rich text representation. |
+| scheme         | string  | Scheme (required)     | Values: S3, GCS, URL, OTHER                                  |
+| id             | string  | Identifier (required) | Host-specific identifier such as an URL or asset id.         |
+| region         | string  | Region                | Provider specific region where the data is stored.           |
+| requester_pays | boolean | Requester pays        | `true` if requester pays, `false` if host pays.              |
+
+### Link Object
+
+TODO: Should be compatible with STAC or remove if catalog is revised.
+
+| Element | Type   | Name                           | Description                         |
+| ------- | ------ | ------------------------------ | ----------------------------------- |
+| href    | string | Hyperlink reference (required) |                                     |
+| rel     | string | Relation (required)            |                                     |
+| type    | string | MIME-type                      | MIME-type of the referenced entity. |
+| title   | string | Title                          | Human-readable title for the link.  |
+
+##  EO extension (eo) - Items and Datasets
+
+TODO
+
+We follow the STAC EO extension, but propose additional fields:
+
+| Element      | Type     | Name               | Description           |
+| ------------ | -------- | ------------------ | --------------------- |
+| unit         | ?        |                    |                       |
```

Comment:
If some sort of unit field is provided it should probably be per-band. But I'm not convinced it's needed, what are the possible values here?

Reply:
Yes, makes sense to have it per band (and per dimension). Moved it to the bands. An example would be °C for temperatures, but could be any unit of measurement. Preferably SI, but could also link to [UDUNITS](https://www.unidata.ucar.edu/software/udunits/) or [the dictionary of UoM](https://www.unc.edu/~rowlett/units/).